ingestion/src/kldata.rs

```diff
@@ -395,7 +405,7 @@ pub async fn filter_and_label_kldata<'a>(
                                 &chunk.station_id,
                                 &param.id,
                                 &chunk.type_id,
-                                &lvl,
+                                &level,
                                 &sensor,
```

Comment:
@intarga this handles the obsinn ingestion part?

Reply:
Yes!